backfiller/backfiller/main.py

```diff
@@ -262,6 +332,9 @@ def run(self):
 			if self.run_once:
 				break
 
+			if self.delete_old and self.start:
+				self.delete_hours()
```

Comment:
if the DB is down / get_nodes() is erroring, deletes won't occur.
this isn't a problem, but it should be noted somewhere. even just a comment here is fine.